cmd/allocate/allocate.go

```diff
@@ -80,7 +80,7 @@ func (c *allocateCommand) Init(args []string) error {
 	var err error
 	c.Budget, c.Limit, err = parseBudgetWithLimit(budgetWithLimit)
 	if err != nil {
-		return err
+		return errors.Annotate(err, `expected args in the form "service budget:limit"`)
```

Comment:
and it should be the other way around: 
expected args in the form "budget:limit service"